src/JuMP.jl

```diff
@@ -517,18 +518,18 @@ Base.copy(c::SDPConstraint, new_model::Model) =
 ##########################################################################
 # ConstraintRef
 # Reference to a constraint for retrieving solution info
-immutable ConstraintRef{T<:JuMPConstraint}
-    m::Model
+immutable ConstraintRef{M<:AbstractModel,T<:JuMPConstraint}
```

Comment:
I considered it, but then I'd have to define my own IainConstraintRef that has exactly the same fields as ConstraintRef, which suggested to me it was the wrong abstraction.